services/arv-git-httpd/git_handler_test.go

```diff
@@ -41,13 +41,14 @@ func (s *GitHandlerSuite) TestEnvVars(c *check.C) {
 }
 
 func (s *GitHandlerSuite) TestCGIError(c *check.C) {
+	log.Printf("git_handler_test: TestCGIError() - expecting a SplitHostPort error, so it is safe to ignore!")
```

Comment:
I was emulating other behaviour I observed when running the tests. While trying to get the dev/test environment working, I very much appreciated tests that told me to expect a failure/warning message. I think the particular one I am thinking of is in the install phase regarding it being safe to ignore package missing warnings.